server/src/main/resources/application.conf

```diff
@@ -164,4 +164,13 @@ feeds {
       }
     }
   }
+  birmingham {
+    soap {
+      connection_timeout = 25000
```

Comment:
our convention so far has been to inline these values where they are used rather than putting them into config, the argument is that you are most likely to want to know about the values when you are looking at the code, it's certainly up for debate as to whether this is a good idea, but probably better to keep it consistent for now.